examples/checkpointing/checkpoint.py

```diff
@@ -0,0 +1,65 @@
+from pyrevolve import Checkpoint, Operator
+from devito import TimeFunction
+
+
+class DevitoOperator(Operator):
+    """Class to wrap devito operators so they conform to the pyRevolve API
+    """
+    def __init__(self, op, args, argnames):
+        self.op = op
+        self.args = args
+        self.argnames = argnames
```

Comment:
can't the argname be obtained directly from the arg?

Reply:
Changed so it becomes more clear